style.css

```diff
@@ -0,0 +1,461 @@
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.youtube-logo {
+    width: 10%;
+}
+
+i {
+    font-size: 150%;
+}
+
+#nav-bar{
+    height: 56px;
+    display: flex;
+    box-shadow: 0px 0px 0px 3px rgba(226, 226, 226, 0.25);
+}
+
+#nav-bar > ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    position: fixed;
+    overflow: hidden;
+    background: rgb(255, 255, 255);
+    width: 100%;
+    display: flex;
+}
+
+#menu-logo {
+    float: left;
+}
+
+.right {
+    float: right;
+}
+
+input {
+    height: 25px;
+    font-size: 20px;
+    font-family: 'Times New Roman', Times, serif;
+}
+
+#nav-search {
+    text-align: center;
+}
+
+#nav-icons {
+text-align: right;
+}
+
+li{
+    list-style-type: none;
+}
+
+.navbar{
+    background-color: rgba(255, 255, 255, 0.945);
+    box-shadow: 0px 0px 0px 3px rgba(226, 226, 226, 0.25);
+    height: 50px;
+}
+
+.navbar-nav.navbar-center {
+    position: absolute;
+    left: 50%;
+    transform: translatex(-50%);
+}
+
+.form-control {
+    margin-top: -2px;
+    pointer-events: none;
+    border-radius: 2px;
+    border-top-right-radius: 0px;
+    border-bottom-right-radius: 0px;
+}
+
+.search-button{
+    height: 33.5px;
+    width: 5.5%;
+    margin-left: -5px;
+    border: 1px solid #929292;
+    border-radius: 2px;
+    border-top-left-radius: 0px;
+    border-bottom-left-radius: 0px;
+    background-color: #e6e6e6;
+}
+
+.search-button > i {
+    padding: 3px;
+    font-size: 15px;
+    margin: 0 auto;
+}
+
+a {
+    color: black;
+}
+
+a:hover {
+    color: black;
+    background-color: white;
+}
+
+.nav>li>a:focus, .nav>li>a:hover {
+    text-decoration: none;
+    background-color: rgba(255, 255, 255, 0);
+    } 
+
+.sign-in-padding{
+    margin-top: -5px;
+}
+
+#sign-in {
+    padding: 5px 20px;
+    color: blue;
+    background-color: white;
+    border-color: blue;
+    pointer-events: none;
+    margin-top: -5px;
+
+}
+
+#sign-in:hover {
+    background-color: rgba(255, 255, 255, 0);
+    box-shadow: none;
+    pointer-events: none;
+}
+
+#media{
+    padding-top: 65px;
+}
+
+.title{
+    margin: 0;
+    padding-left: 135px;
+    padding-right: 135px;
+    height: 80px;
+    width: 100%;
+    margin-bottom: 15px;
+}
+
+h1{
+    font-size: 20px;
+    font-weight: bold;
+    margin-bottom: 2px;
+    text-align: left;
+}
+
+#views{
+    color: rgb(100, 100, 100);
+    text-align: left;
+}
+
+#video-options{
+    justify-content: space-between;
+    display: flex;
+    align-items: center;
+    font-size: 11px;
+    color: #606060;
+    width: 25%;
+    float: right;
+}
+
+#video-options > i{
+    margin: 2px;
+}
+
+#description {
+    padding-left: 80px;
+}
+
+#description{
+    width: 100%;
+    vertical-align: middle;
+    padding-left: 135px;
+    padding-right: 135px;
+}
+
+section{
```

Comment:
Remove these comments if they are not needed. Don't leave unused styles as comments in you code again.